src/third_party/rapier/shaders/instancing3d.wgsl

```diff
@@ -21,10 +17,16 @@ struct VertexOutput {
 @vertex
 fn vertex(vertex: Vertex) -> VertexOutput {
     let position = vertex.position * vertex.i_pos_scale.w + vertex.i_pos_scale.xyz;
-    let world_position = mesh.model * vec4<f32>(position, 1.0);
 
     var out: VertexOutput;
-    out.clip_position = view.view_proj * world_position;
+    // NOTE: Passing 0 as the instance_index to get_model_matrix() is a hack
+    // for this example as the instance_index builtin would map to the wrong
+    // index in the Mesh array. This index could be passed in via another
+    // uniform instead but it's unnecessary for the example.
+    out.clip_position = mesh_position_local_to_clip(
+        get_model_matrix(0u),
+        vec4<f32>(position, 1.0)
+    );
```

Comment:
From my understanding, this code is the culprit for "jumping" rendering descirbed in the PR description.

if I input `2u` to `get_model_matrix` it seems to work correctly. Now I'm not sure how I should provide that value.

I've tried to leverage `@builtin(instance_index) instance_index: u32,` but this seems to be an ever increasing value which doesn't help in my case.

Reply:
I'm now passing identity